recipes/rdflib/meta.yaml

```diff
@@ -0,0 +1,75 @@
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set name = "rdflib" %}
+{% set version = "4.2.1" %}
+{% set sha256 = "eb02bd235606ef3b26e213da3e576557a6392ce103efd8c6c8ff1e08321608c8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  # By default, the package will be built for Python 2.7, 3.4, and above, for all major OSs.
+  # Add the line "skip: True  # [py<34]" (for example) to limit to Python 3.4 and newer, or "skip: True  # [not win]" to limit to Windows.
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - isodate
+    - pyparsing
+
+  run:
+    - python
+    - isodate
+    - pyparsing
+
+test:
+  # Python imports
+  imports:
+    - rdflib
+    - rdflib/extras
+    - rdflib/plugins
+    - rdflib/plugins/parsers
+    - rdflib/plugins/parsers/pyMicrodata
+    - rdflib/plugins/parsers/pyRdfa
+    - rdflib/plugins/parsers/pyRdfa/extras
+    - rdflib/plugins/parsers/pyRdfa/host
+    - rdflib/plugins/parsers/pyRdfa/rdfs
+    - rdflib/plugins/parsers/pyRdfa/transform
+    - rdflib/plugins/serializers
+    - rdflib/plugins/sparql
+    - rdflib/plugins/sparql/results
+    - rdflib/plugins/stores
+    - rdflib/tools
+
+  commands:
+    # You can put test commands to be run here.  Use this to test that the
+    # entry points work.
+
+    - rdfpipe --help
+    - csv2rdf --help
+    - rdf2dot --help
+    - rdfs2dot --help
+    - rdfgraphisomorphism --help
+
+  requires:
+    - html5lib
+    - networkx
+
+about:
+  home: https://github.com/RDFLib/rdflib
+  license: BSD License
```

Comment:
Please specify the license version (e.g. `BSD 3-Clause` if it is the 3 -Clause variant).